src/main/kotlin/lotto/ui/InputView.kt

```diff
@@ -30,7 +30,23 @@ class InputView {
         println("보너스 볼을 입력해 주세요.")
         return readLine()!!.trim()
     }
-    
+
+    fun getManualTicketCount(): Int {
+        println("수동으로 구매할 로또 수를 입력해 주세요.")
+        return readLine()!!.trim().toInt()
+    }
+
+    fun getManualTicketNumbers(count: Int): MutableList<List<Int>> {
+        println("수동으로 구매할 번호를 입력해 주세요.")
+        val numbers = mutableListOf<List<Int>>()
+        IntRange(1, count).forEach {
+            val input = readLine()!!
+            numbers.add(input.split(SEPERATOR).map { it.toInt() })
+        }
+
+        return numbers
+    }
```

Comment:
컬렉션은 불변 타입으로 리턴하는게 좋아요~
```suggestion
     fun getManualTicketNumbers(count: Int): List<List<Int>> {
        println("수동으로 구매할 번호를 입력해 주세요.")
        return IntRange(1, count).map {
            val input = readLine()!!
            input.split(SEPERATOR).map { it.toInt() }
        }
    }

   
```